README.md

```diff
@@ -4,8 +4,9 @@ A package of resources for Hypothesis front-end applications.
 
 #### Requirements
```

Comment:
The heading levels are a bit wonky here. We've got a level 1 heading and underneath that a level 4 sub-heading ("Requirements") followed by a level 3 heading ("Usage"). I would correct this so that there is a level 1 heading at the top with level 2 for the next logical level down and so on.